content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -201,8 +201,8 @@ For a reference to old feature gates that are removed, please refer to
 | `TopologyAwareHints` | `false` | Beta | 1.23 | 1.23 |
 | `TopologyAwareHints` | `true` | Beta | 1.24 | |
 | `TopologyManagerPolicyAlphaOptions` | `false` | Alpha | 1.26 | |
-| `TopologyManagerPolicyBetaOptions` | `false` | Beta | 1.26 | |
-| `TopologyManagerPolicyOptions` | `false` | Alpha | 1.26 | |
+| `TopologyManagerPolicyBetaOptions` | `true` | Beta | 1.26 | |
```

Comment:
```suggestion
| `TopologyManagerPolicyBetaOptions` | `true` | Beta | 1.26 | 1. 27 |
```

Reply:
changed

Reply:
No. This gate will stay there as Beta for a long time.